core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java

```diff
@@ -94,7 +94,10 @@ private void handleBranchCommit(long msgId, String serverAddress,
             resultMessage = (BranchCommitResponse)handler.onRequest(branchCommitRequest, null);
             sender.sendResponse(msgId, serverAddress, resultMessage);
         } catch (Exception e) {
-            LOGGER.error(FrameworkErrorCode.NetOnMessage.errCode, e.getMessage(), e);
+            LOGGER.error(FrameworkErrorCode.NetOnMessage.getErrCode(), e.getMessage(), e);
+            if(resultMessage == null){
```

Comment:
maybe resultMessage already is not null

Reply:
> maybe resultMessage already is not null

`handler.onRequest` may throw a `RuntimeException`,then `resultMessage` will be null.

Reply:
> > maybe resultMessage already is not null
> 
> `handler.onRequest` may throw a `RuntimeException`,then `resultMessage` will be null.

yes